src/encodeExperiment.js

```diff
@@ -0,0 +1,58 @@
+import { createXArray } from 'ml-spectra-processing';
+
+export function encodeExperiment(array) {
+  let returnArray = [];
+  let dict = {};
+  let rowNumber = 0;
+
+  for (let i = 0; i < array.length; i++) {
+    let returnRow;
+
+    if ('min' in array[i]) {
+      let label = array[i]['label'];
+      let min = array[i]['min'];
+      let max = array[i]['max'];
+      let nbSamples = array[i]['nbSamples'];
+      let sampling = array[i]['sampling'];
```

Comment:
```suggestion
      let sampling = array[i].sampling;
```

Same should be done everywhere